envs/uwsgi-env.yaml

```diff
@@ -0,0 +1,7 @@
+builder_version: ">=13.0.0"
+
+packages:
+  - feedstock: https://github.com/conda-forge/uwsgi-feedstock
```

Comment:
Pls move this feedstock in opence-env.yaml.

Reply:
Moved to opence-env